segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -947,11 +948,16 @@ private Void handleException(long requestId, String segment, long offset, String
             invokeSafely(connection::send, new SegmentRead(segment, offset, true, false, EMPTY_BYTE_BUFFER, requestId), failureHandler);
         } else if (u instanceof CancellationException) {
             log.info(requestId, "Closing connection {} while performing {} due to {}.",
-                     connection, operation, u.toString());
+                    connection, operation, u.toString());
             connection.close();
+        } else if (u instanceof TokenExpiredException) {
+            log.warn(requestId, "Token was found as expired during operation '{}'.", operation, u);
```

Comment:
1. "Expired token during operation `{}`"
2. We don't care about the full stack trace here. Do not log `u`.

Reply:
Fixed.